gov2/secretsmanager/README.md

```diff
@@ -0,0 +1,48 @@
+# AWS SDK for Go V2 code examples for AWS Secrets Manager 
+
+## Purpose
+
+These examples demonstrate how to perform several AWS Secrets Manager
+operations using version 2 of the AWS SDK for Go.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
+
+## Running the code
+
+To run the full example suite, enter:
+
+```
+go run main.go
+```
+
+This example will
+
+* Create a secret (with only an ARN to identify it)
+* Get the value of that secret
+* Change the value of the secret
+* Show the change has taken effect
+* List secrets in your account by their ARNs
+* Delete the secret that was created
+
+### Notes
+
+- We recommend that you grant this code least privilege,
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
+
+
```

Comment:
Include a section ##Unit tests. If the process for running unit tests is similar for all your examples across all services, you could just link to a ##Unit tests sections at the language level, and describe the process there.

Include a section ##Contributing - To propose a new code example to the AWS documentation team, see the CONTRIBUTING.md (https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/CONTRIBUTING.md). The team prefers to create code examples that show broad scenarios rather than individual API calls. 

Consider adding a 'Resources' section with (for example), links to the the Go SDK user guide, the Go API reference guide, etc.

Reply:
At the moment, there are no unit tests for this set (doing so is tedious at best)

I've included a resources section however.